packages/block-editor/src/components/media-placeholder/index.js

```diff
@@ -74,7 +75,6 @@ export function MediaPlaceholder( {
 	onToggleFeaturedImage,
 	onDoubleClick,
 	onFilesPreUpload = noop,
-	onHTMLDrop = noop,
```

Comment:
We should probably keep this as a fallback and "deprecate it" for folks that are defining this prop.

Reply:
Yeah, it's a todo that I don't include in this draft state but will do later 👍.

This prop is never documented though, I'm curious if we should maintain full backward-compatibility in this case, or could we just deprecate it with a no-op?